TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -96,7 +96,7 @@ public class Scanner : AbstractScanner
 
             // Comment line => return only one token with type CommentLine
             // Debug line => treated as a comment line if debugging mode was not activated
-            if (textLine.Type == CobolTextLineType.Comment || (tokensLine.Type == CobolTextLineType.Debug && !IsDebugLineActive(tokensLine)))
+            if (textLine.Type == CobolTextLineType.Comment || (tokensLine.Type == CobolTextLineType.Debug && !IsDebuggingModeActive(tokensLine.ScanState, tokensLine.SourceText)))
```

Comment:
If I understand correctly, the scanState of following lines of a continuation group is null, that's why we need to use only the scanState of the first line of a continuation group.
If so, I'd rather make sure to always have a non null scanState on all lines of a continuation group to avoid this bug in other situations.

Reply:
No it's not that.

When scanning a continuation group, the scanState of the line is set after the test on the debugging mode. Here we can use directly the property of the line when calling the method but in the continuation group case, we pass the instance of scanState that would be set later on the tokens line.

Reply:
Added a comment for that special case